integration/lsp/test_utils.ts

```diff
@@ -73,6 +73,7 @@ export function initializeServer(client: MessageConnection): Promise<lsp.Initial
           }
         },
         moniker: {},
+        definition: {linkSupport: true}
```

Comment:
I updated all existing tests to report as a client that supports `linkSupport`, as to not have the two existing "Go To Definition" tests failing.

Reply:
👍